.eslintrc.js

```diff
@@ -2,55 +2,46 @@
 /* eslint quotes: ["error", "double"] */
 
 module.exports = {
-  "env": {
-    "node": true,
-    "es6": true,
-    "jest": true,
+  env: {
+    node: true,
+    es6: true,
+    jest: true,
   },
-  "plugins": ["lodash"],
-  "extends": ["airbnb-base", "plugin:lodash/recommended"],
-  "rules": {
-    "array-bracket-newline": ["error", { "multiline": true }],
-    "func-names": "off",
-    "no-underscore-dangle": "off",
-    "function-paren-newline": "off",
-    "indent": [
-      "error",
-      2,
-      {
-        "CallExpression": { "arguments": "first" },
-        "FunctionDeclaration": { "parameters": "first" },
-        "FunctionExpression": { "parameters": "first" },
-      },
-    ],
-    "lodash/prefer-lodash-method": [0],
-    "lodash/prefer-noop": [0],
-    "max-len": ["error", { "code": 110 }],
-    "operator-linebreak": ["error", "before"],
+  plugins: ['lodash', 'prettier'],
+  extends: ['airbnb-base', 'plugin:lodash/recommended', 'prettier'],
+  rules: {
+    'func-names': 'off',
+    'function-paren-newline': 'off',
+    'lodash/prefer-lodash-method': [0],
+    'lodash/prefer-noop': [0],
+    'max-len': ['error', { code: 110 }],
+    'no-underscore-dangle': 'off',
+    'prettier/prettier': 'error',
```

Comment:
Avoid changing formatting and config in the same commit unless syncing configs.
Also, this violates the Eslint rule overrides at the top of the file.

I kept double quotes as their documentation uses that and many projects do too, so it's easier to copy rules as-is.